spec.html

```diff
@@ -42583,6 +42951,32 @@ <h1>
           1. Return *true*.
         </emu-alg>
       </emu-clause>
+
+      <emu-clause id="sec-hostgrowsharedarraybuffer" type="host-defined abstract operation">
+        <h1>
+          HostGrowSharedArrayBuffer (
+            _buffer_: a SharedArrayBuffer,
+            _newByteLength_: a non-negative integer,
```

Comment:
`newByteLength` can actually be negative and ±Infinity because currently `ToIntegerOrInfinity` is used to convert the input argument. When switching to `ToIndex`, this definition will be correct, though.

Reply:
Done.